src/Benchmark/UdpStatSendingBenchmark.cs

```diff
@@ -32,7 +32,7 @@ public void Setup()
                 config.Port,
                 config.DnsLookupInterval);
 
-            _transport = new UdpTransport(endpointSource);
+            _transport = new SocketTransport(endpointSource, SocketProtocol.Udp);
```

Comment:
Is there any utility in making an overload that just takes the IP endpoint and defaults to `Udp` for the common case?

Reply:
I don't see a need for it, I would not recommend using that internally